tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/ActivityController.java

```diff
@@ -249,7 +249,8 @@ public Response forgetWork(@HeaderParam(LRA_HTTP_HEADER) String lraId) {
 
     @PUT
     @Path(ActivityController.ACCEPT_WORK)
-    @LRA(LRA.Type.REQUIRED)
+    @LRA(value = LRA.Type.REQUIRED, terminal = false)
+    // TODO Verify, this seems not used by any of the TCK tests
```

Comment:
valid, only used in one test that explicitly closes/compensates started LRA

Reply:
The method is invoked by the [test that validates](https://github.com/eclipse/microprofile-lra/blob/master/tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java#L565) that if the compensate callback method returns the `202 Accepted` HTTP status code then the implementation behaves correctly.